### PR TITLE
Implemented logout test

### PR DESCRIPTION
**Pull request by Schlvf**
## Changes: 
- Implemented logout test case
- Removed unnecessary using

### CODE REVIEW

File: 3.Tests/UI/StepDefinitions/User/LogoutSteps.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    public LogoutStepDefinitions(ScenarioContext scenarioContext) : base(scenarioContext)
+    {
+        _scenarioContext = scenarioContext;
+        _homePage = new HomePage();
+    }
+
+    [AfterScenario]
+    public void TearDown()
+    {
+        GenericWebDriver.Dispose();
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/StepDefinitions/User/LogoutSteps.cs:24]: Move this from the steps class to a hooks class, we shouldn't use NUnit here

File: 3.Tests/UI/StepDefinitions/User/LogoutSteps.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+﻿using SeleniumTest.Core.Drivers;
+using SeleniumTest.Tests.Steps.Commons;
+using TechTalk.SpecFlow;
+using Views.WebAppPages;
+
+namespace SeleniumTest.Tests.Steps.User;
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/StepDefinitions/User/LogoutSteps.cs:6]: Pages and Tests projects should have the AUT prefixed on the namespace: `namespace todoly.Tests.Steps.User;`

File: 3.Tests/UI/StepDefinitions/User/LogoutSteps.cs (hunk 1 above)
Comment (Schlvf) [reply 3.Tests/UI/StepDefinitions/User/LogoutSteps.cs:25]: We don't have a hook class yet and the `TearDown` method is not a NUnit method, its just the name that the after scenario method has thats using the Specflow package

Comment (sebasonti-jalauni) [reply 3.Tests/UI/StepDefinitions/User/LogoutSteps.cs:25]: You're right, my bad, I confused it with a TearDown method from NUnit.

**Review by sebasonti-jalauni — APPROVED** [review]

**Review by gioacchinogioffre — APPROVED** [review]

**Pull request merged**